libr/core/cmd.c

```diff
@@ -2445,6 +2445,10 @@ static int r_core_cmd_subst(RCore *core, char *cmd) {
 	 * nested call of this function */
 	ut64 orig_offset = core->offset;
 	icmd = strdup (cmd);
+	if (!icmd) {
+		eprintf ("Command line too long\n");
```

Comment:
i wont consider "command line too long" as a result of a failed strdup, probably better to remove the eprintf

Reply:
ok 98d10fc4a8c9241da103e892614c5291577a233d